pkg/netceptor/netceptor.go

```diff
@@ -932,7 +935,7 @@ func (s *Netceptor) GetServerTLSConfig(name string) (*tls.Config, error) {
 // AddWorkCommand records a work command so it can be included in service announcements.
 func (s *Netceptor) AddWorkCommand(command string, secure bool) error {
 	if command == "" {
-		return fmt.Errorf("must provide a name")
+		return errors.New(errMustProvideName)
```

Comment:
Hmmm. I dislike that all these error messages are identical. Can we take this opportunity to mention **_which_** name is missing?
```suggestion
		return errors.fmt(errMustProvideName,"Work Command")
```